vowpalwabbit/slim/test/ut_opts.cc

```diff
@@ -4,15 +4,15 @@
 using namespace ::testing;
 using namespace vw_slim;
 
-TEST(CommandLineOptions, parsing)
+TEST(command_line_options_slim, parsing)
```

Comment:
If you change this you also need to update this: https://github.com/VowpalWabbit/vowpal_wabbit/blob/5c920b44f3f554820ff1fce01a7976dd1907814f/.github/workflows/build_vw_slim.yml#L31